Console/Command/RemoveUnusedAttributesCommand.php

```diff
@@ -45,17 +45,27 @@ protected function configure()
         $this
             ->setName('eav:attributes:remove-unused')
             ->setDescription('Remove unused attributes (without values or not assigned to any attribute set')
-            ->addOption('dry-run');
+            ->addOption('dry-run')
+            ->addOption('force');
     }
 
     public function execute(InputInterface $input, OutputInterface $output)
```

Comment:
We could add `int` as a return type now.

Reply:
Thanks for catching this. I've updated this now as intended initially.